x-pack/plugins/endpoint/public/embeddables/resolver/models/indexed_process_tree.ts

```diff
@@ -5,15 +5,16 @@
  */
 
 import { uniquePidForProcess, uniqueParentPidForProcess } from './process_event';
-import { IndexedProcessTree, ProcessEvent } from '../types';
+import { IndexedProcessTree } from '../types';
+import { EndpointEvent } from '../../../../common/types';
```

Comment:
❔ per @andrewstucki we will end up with two types

Reply:
Yep, looks like this interface is currently describing legacy endgame events, while our focus should be on the ECS-specified elastic-native events. Ideally we can do both to support both the legacy platform and new endpoint product through resolver, and just union their types together--but yeah, separate PR for that sometime in the future is most welcome.

Reply:
Yeah maybe eventually importing `ResolverEvent` and doing some type guard to narrow the type to either a `LegacyEndpointEvent` or `EndpointEvent`.